components/rmrk/Gallery/Holder/Holder.vue

```diff
@@ -220,13 +238,23 @@ type BaseTableRow = {
   Amount: number
 }
 
-@Component({ components })
+@Component({
+  filters: {
+    percentageFilter: (value) => {
```

Comment:
toPercent already exist

Reply:
Okay. And the 'toPercent' method needs to do some rework